.github/workflows/R.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: R
+on:
+  workflow_call:
+    inputs:
+      do-lint:
+        description: 'Flag to run `lintr`'
+        required: false
+        default: true
+        type: boolean
+      do-codecov:
+        description: 'Flag to check codecoverage and upload to codecov.io'
+        required: false
+        default: true
+        type: boolean
```

Comment:
```suggestion
on:
  push:
    branches:
      - main
  pull_request:
```

Reply:
Missed this earlier. These are the actual triggers that make the workflow "go". then the `workflow_call` is in the `RMI-PACTA/actions` repo (since that's the one being called).